internal/controller/authconfigs_reconciler.go

```
@@ -55,6 +60,75 @@ func (r *AuthConfigsReconciler) Subscription() controller.Subscription {
+				errorRegistry.Record(
+					AuthConfigsReconcilerName,
+					OperationDelete,
+					k8stypes.NamespacedName{Name: ac.GetName(), Namespace: ac.GetNamespace()},
+					kuadrantauthorino.AuthConfigGroupKind,
+					err,
+				)
+			}
+		}
+
+		logger.Info("auth configs deleted, removing finalizer from Kuadrant CR")
+		_, err := r.ReconcileResource(ctx, &kuadrantv1beta1.Kuadrant{}, kObj.DeepCopy(), func(existing, desired client.Object) (bool, error) {
```

golangci-lint (2.12.2)

[warning] unused-parameter: parameter 'desired' seems to be unused, consider removing or renaming it as _

(revive)